src/polyfill/RTCDataChannel.ts

```diff
@@ -1,26 +1,28 @@
 import 'node-domexception';
-import * as exceptions from './Exception.js';
-
-export default class _RTCDataChannel extends EventTarget {
-    #dataChannel;
-    #readyState;
-    #bufferedAmountLowThreshold;
-    #binaryType;
-    #maxPacketLifeTime;
-    #maxRetransmits;
-    #negotiated;
-    #ordered;
+import * as exceptions from './Exception';
+import { DataChannel } from '../lib/index';
+
+export default class RTCDataChannel extends EventTarget {
```

Comment:
I'd do:
`npm i @types/webrtc -D`
and
```ts
/// <reference types="@types/webrtc" />
export default class RTCDataChannel extends EventTarget implements globalThis.RTCDataChannel {
```

Reply:
I will check that when I have time for this PR.